aws/logs_monitoring/tests/test_parsing.py

```diff
@@ -233,6 +237,242 @@ def test_s3_source_if_none_found(self):
         self.assertEqual(parse_event_source({"Records": ["logs-from-s3"]}, ""), "s3")
 
 
+class TestParseAwsWafLogs(unittest.TestCase):
```

Comment:
Can we add two more cases here where event comes in as a string:

- valid JSON
- invalid JSON